### ISSUE TITLE
project doesn't support -o option

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1600]
In most cases, I can use **-F** option to specify the output of **project** command.

However, if **-G** is used, I cann't use **-F** any more.
```
$ gmt project -C0/0 -A30 -G10 -L0/60 -Frs  
project [ERROR]: Syntax error -G option: -F not allowed [Defaults to rsp]
```
Any reason why **project** doesn't support **-o** option?

**Comment by PaulWessel** [comment]
I think it is historical in that -G produces r,s,p only so all the other letters cannot be used anyway.  You can get what you want via -o but we could probably figure out how to use -F to be consistent with the non-G mode.  You could make this a feature request.

**Comment by PaulWessel** [comment]
Sorry, completely missed your point about -o. Yes, I think -o should be allowed - it is universal.  Try to add o to the THIS_MODULE_OPTIONS in project.c and see if it works (then add to usage syntax, synopsis, man page too of course).

**Comment by seisman** [comment]
OK, I'll open a PR later.

**Comment by seisman** [comment]
Closed by #1603.
